src/db/collections/outputs/indexer/mod.rs

```diff
@@ -392,10 +392,10 @@ impl OutputCollection {
 
         self.create_index(
             IndexModel::builder()
-                .keys(doc! { "metadata.booked": -1 })
+                .keys(doc! { "metadata.booked.milestone_index": -1 })
                 .options(
                     IndexOptions::builder()
-                        .name("output_booked_index".to_string())
+                        .name("output_booked_milestone_index".to_string())
                         .partial_filter_expression(doc! { "metadata.spent_metadata": null })
```

Comment:
We should really also remove this filter. None of our queries that use this index specify it.